controller/app.controller.js

```diff
@@ -39,4 +40,22 @@ const getArticles = (req, res, next) => {
   });
 };
 
-module.exports = { getTopics, getAll, getArticleById, getArticles };
+const getCommentsByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+  selectCommentsByArticleId(article_id)
+    .then((comments) => {
+      if (comments.length === 0) {
+        res.status(404).send({ msg: "No comments found for this article!" });
+      }
```

Comment:
once you add your exists but doesnt have any comments test youll have to add some more functionality :)